### PR TITLE
feat: show error in Lua if ELRS module not enabled

### PR DESCRIPTION
**Pull request by pfeerick**
For consideration. 

Rather than just sitting there dumbly saying "Loading...", tell the user they need to enable an ELRS module ;)

Also function return value indicates which modules are enabled, as a bonus, if becomes useful in the future for any reason.

Simulator: 
screenshot_tx16s_24-07-10_11-22-06

Radio: 
image

image

### CODE REVIEW

**Comment by CapnBry** [comment]
I like the idea, but don't want it to run on every loop iteration and might want to massage the message a bit since there is no "ELRS" in model settings, mind if I commit to this branch?

**Comment by pfeerick** [comment]
Of course you may! ;) I was thinking about making it single shot check, but thought I'd see what the initial reaction was before investing time in any improvements.

**Comment by CapnBry** [comment]
Woof. I can't believe I spent 3 hours on this, but inconsistent behavior of `error()`, `popupConfirmation()` combined with b&w and colorlcd, and multiple versions of EdgeTX / OpenTX made it a real nightmare. Trying to do a multiline message absolutely impossible due to linebreaks being handled all sorts of differently across versions and platforms. I eventually broke down and wrote it using `lcd.drawText()`

The way I implemented it adds a single loop delay for everyone before the script starts running, but what is that, 20ms? The idea being to reduce the amount of code that doesn't get thrown out to reduce normal running RAM.

Here's how it looks, and it looks the same on old OpenTX too
image

### Tested

* RadioMaster TX16S internal, EdgeTX 2.10.1
* RadioMaster Ranger Micro external EdgeTX 2.10.1
* TX16S EdgeTX 2.10.1 no module enabled
* RadioMaster Boxer internal EdgeTX 2.10.0-rc3
* Boxer EdgeTX 2.10.0-rc3 no module enabled
* SIYI FM30 external OpenTX 2.3.14
* OpenTX 2.3.14 no module enabled
* BetaFPV Lite Radio Pro internal EdgeTX 2.8.0-selfbuild -- Loads the whole lua, but crashes OOM on refresh. Runs fine on subsequent runs
* Lite Radio Pro EdgeTX 2.8.0 no module enabled

**Review by pkendall64 — APPROVED** [review]

File: src/lua/elrsV3.lua [hunk 1]
```diff
@@ -891,6 +891,31 @@ local function setMock()
...
+  -- No CRSF module found, save an error message for run()
+  lcd.clear()
+  local y = 2
+  lcd.drawText(2, y, "  No ExpressLRS", MIDSIZE)
+  y = y + (textSize * 2)
+  lcd.drawText(2, y, " Enable a CRSF Internal")
+  y = y + textSize
+  lcd.drawText(2, y, "   or External module in")
+  y = y + textSize
+  lcd.drawText(2, y, "      Model settings")
```
Comment (mha1) [inline src/lua/elrsV3.lua:914]: Internal modules are enabled in Radio settings / Hardware :grimacing:

**Comment by pfeerick** [comment]
Oh, very nice! :) Yeah... I really didn't want to go down the rabbit hole of trying to make it look nice on both B&W and colorlcd, OTX/ETX, etc... so my condolences and congratulations on making it look this pretty! :rofl:

File: src/lua/elrsV3.lua (hunk 1 above)
Comment (pfeerick) [reply src/lua/elrsV3.lua:914]: Well, yes and no... they are *usually* configured once off in Radio Settings/SYS => Hardware... but other than that... it's Model Settings => ... ie. OFF/CRSF (ON). If it's wrong there, that is a different issue... RTFM is required :rofl:

Comment (CapnBry) [reply src/lua/elrsV3.lua:914]: Do we think this is too much?
image

Comment (mha1) [reply src/lua/elrsV3.lua:914]: I'd say it is correct and fits.

Comment (CapnBry) [reply src/lua/elrsV3.lua]: Great! I'mma recuse myself due to how much I've taken over and let you do the approval if you don't mind :)

**Comment by CapnBry** [comment]
> Oh, very nice! :) Yeah... I really didn't want to go down the rabbit hole of trying to make it look nice on both B&W and colorlcd, OTX/ETX, etc... so my condolences and congratulations on making it look this pretty! 🤣

Thanks! I appreciate you bringing this to the top of my todo list, and with already working code too? I'm livin' the dream over here.

**Pull request merged**